tests/api2/test_011_user.py

```diff
@@ -273,7 +273,7 @@ def test_24_look_user_is_delete(request):
 
 def test_25_has_root_password(request):
     depends(request, ["user_02", "user_01"])
-    assert GET('/user/has_root_password/', anonymous=True).json() is True
+    assert GET('/user/has_local_administrator_set_up/', anonymous=True).json() is True
```

Comment:
Should this be account installed of user?

Reply:
No I checked in the api docs, user is corrected. tested in postman before updating